qemu/tests/cfg/seabios.cfg

```diff
@@ -8,19 +8,56 @@
     boot_menu = on
     enable_sga = yes
     image_verify_bootable = no
+    restart_key = "ctrl-alt-delete"
     boot_menu_key = "f12"
     Host_RHEL.m7:
         boot_menu_key = "esc"
     boot_menu_hint = "Press .*(F12|ESC) for boot menu"
-    # Specify the boot device name which you want to test here.
-    boot_device = "iPXE"
-    Host_RHEL.m6:
-        boot_device = "gPXE"
-    # SGA Bios info message, using sep as ";"
-    # Please update this message to suit for your own system.
-    restart_key = "ctrl-alt-delete"
-    Host_RHEL:
-        sgabios_info = "Google, Inc.\s"
-        sgabios_info += "Serial Graphics Adapter .*\s"
-        sgabios_info += "SGABIOS \$Id: sgabios.S .*\s"
-        sgabios_info += "\d \d\s"
+    variants:
+        - reboot_timeout_option:
+            no q35
+            images += " stg"
+            image_name_stg = "images/storage"
+            image_format_stg = "qcow2"
+            drive_format_stg = virtio
+            image_size_stg = 4G
+            create_image_stg = yes
+            remove_image_stg = yes
+            bootindex_stg = 0
+            bootindex_image1 = 1
+            variants:
+                - time0:
+                    boot_reboot_timeout = 0
+                - time1:
+                    boot_reboot_timeout = 1000
+                - time2:
+                    boot_reboot_timeout = 10000
+                - time3:
+                    boot_reboot_timeout = 30000
+                - time4:
+                    boot_reboot_timeout = 60000
+                - time5:
+                    boot_reboot_timeout = 65535
+                - time6:
+                    boot_reboot_timeout = 65536
+                - time7:
+                    boot_reboot_timeout = -1
+        - sga_restart:
+            boot_menu_key = "f12"
+          # SGA Bios info message, using sep as ";"
+          # Please update this message to suit for your own system.
+            restart_key = "ctrl-alt-delete"
```

Comment:
Please recheck `boot_menu_key` and `restart_key`, they have been defined out of first `variants`.